launcher/et

```diff
@@ -32,7 +32,7 @@ do
             shift
             ;;
         -h|--help)
-            printf "et (options) [user@]hostname[:port]
+            printf "et (options) [user@]hostname
```

Comment:
Oops, I see the port info *is* properly propagated, and my commit does eliminate functionality.  Skip it if you don't like it :)